AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockSetConfigActionGroup.xml

```diff
@@ -17,8 +17,7 @@
         </arguments>
         <magentoCLI command="config:set {{AdobeStockConfigDataEnabled.path}} {{enabled}}" stepKey="setModuleEnabled"/>
         <magentoCLI command="config:set {{AdobeStockConfigDataWysiwygEnabled.path}} {{wysiwygEnabled}}" stepKey="enableWYSIWYG"/>
-<!--        TODO: uncomment next lines where issue with sensitive config setting will be resolved. -->
-<!--        <magentoCLI command="config:sensitive:set {{AdobeStockConfigDataApiKey.path}} {{apiKey}}" stepKey="setModuleApiKey"/>-->
-<!--        <magentoCLI command="config:sensitive:set {{AdobeStockConfigDataPrivateKey.path}} {{privateKey}}" stepKey="setModulePrivateKey"/>-->
+        <magentoCLI command="config:set {{AdobeStockConfigDataApiKey.path}} {{apiKey}}" stepKey="setModuleApiKey"/>
```

Comment:
`config:set` does not work properly for values that should be encrypted in the DB and inserts them as is. This approach will not work

Reply:
Hello @sidolov ,

If we move the `{{apiKey}}` from .env files to .credentials as per this [docs](https://devdocs.magento.com/mftf/docs/credentials.html) then we can manage the sensitive data and value is encrypted in MFTF 2.4.3 version. You can check this [link](https://www.youtube.com/watch?v=OHu-r1FtiZE)

Thanks

Reply:
I missed that you also fixed `AdobeStockImageAdminUi/Test/Mftf/Data/AdobeStockConfigData.xml` file. I'll check it, thanks!